### ISSUE TITLE
The requested module '../cache.js' does not provide an export named 'isCached'

### ISSUE THREAD

**Issue by mymikemiller** [rvanasa/canpack#6]
I'm getting the following error when running the `canpack` command

```
file:///Users/mike/.nvm/versions/node/v20.13.1/lib/node_modules/ic-mops/dist/commands/install.js:10
import { addCache, copyCache, isCached } from '../cache.js';
                              ^^^^^^^^
SyntaxError: The requested module '../cache.js' does not provide an export named 'isCached'
```

My test project can be found [here](https://github.com/mymikemiller/s3_presigned_url). Note that I'm not even trying to import any crates yet, but even if I do I get the same error when running `canpack`

Is this tool in a working state right now?

My ultimate goal is to use the functions exposed in [Amazon's S3 rust crate](https://docs.rs/aws-sdk-s3/latest/aws_sdk_s3/) (as described in the Rust section [here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/example_s3_Scenario_PresignedUrl_section.html)) from within my Motoko canister. I'm under the impression that I should be able to essentially import it into a rust canister (using canpack) and call its functions from my own Rust canister, and ultimately from one of my Motoko canisters via an inter-canister query.

If this isn't the right way to go about things, I'm all ears! I was referred to canpack via [this](https://forum.dfinity.org/t/using-rust-packages-from-a-motoko-canister/30055) and this forum post and the [referenced thread](https://forum.dfinity.org/t/exploration-calling-rust-functions-from-motoko/27913) investigating the ability to do this type of thing.

Any and all advice is appreciated, and thanks for your work on this!

**Comment by rvanasa** [comment]
Hi! Thanks for your interest in this library! This appears to be a bug with a previous version of the [`ic-mops`](https://github.com/ZenVoich/mops/) dependency; I will update the `canpack` npm package to hopefully fix this issue. 

Let me know if you continue to encounter this error message on the latest version (v0.2.3). Currently at an airport, so I will work on this as soon as possible.

**Comment by rvanasa** [comment]
Released version 0.2.3, which should fix this error message. I'll reopen the issue if needed. Cheers!
